flexmeasures/ui/templates/crud/account.html

```diff
@@ -0,0 +1,155 @@
+                    <div class="checkbox">
+                        <label>
+                            <input name="include_inactive" {% if include_inactive %} checked="checked" {% endif %}
+                                type="checkbox"> Include inactive
+                        </label>
+                    </div>
+                </form>
+                <table class="table table-striped table-responsive paginate">
+                    <thead>
+                        <tr>
+                            <th>Username</th>
+                            <th>Email</th>
+                            <th>Roles</th>
+                            <th>Account</th>
+                            <th>Timezone</th>
+                            <th>Last Login</th>
+                            <th>Last Seen</th>
+                            <th>Active</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for user in users %}
+                        <tr>
+                            <td>
+                                <a href="/users/{{ user.id }}" title="View/edit this user">{{ user.username
+                                    }}</a>
+                            </td>
+                            <td>
+                                <a href="mailto:{{ user.email }}" title="Mail this user">{{ user.email }}</a>
+                            </td>
+                            <td>{% for role in user.flexmeasures_roles %}
+                                {{ role.name }}{{ "," if not loop.last }}
+                                {% endfor %}
+                            </td>
+                            <td>
+
+                                {{ user.account.name }}
+                            </td>
+                            <td>
+                                {{ user.timezone }}
+                            </td>
+                            <td title="{{  user.last_login_at | localized_datetime }}">
+                                {{ user.last_login_at | naturalized_datetime}}
+                            </td>
+                            <td title="{{  user.last_seen_at | localized_datetime }}">
+                                {{ user.last_seen_at | naturalized_datetime}}
+                            </td>
+                            <td>
+                                {{ user.active }}
+                            </td>
+
+                        </tr>
+                        {% endfor %}
+                    </tbody>
+                </table>
+                <h3>Assets</h3>
+
+                <table class="table table-striped table-responsive paginate">
+                    <thead>
+                        <tr>
+                            <th>Name</th>
+                            <th class="text-right no-sort">Location</th>
+                            <th class="text-right">Asset id</th>
+                            <th class="text-right">Account</th>
```

Comment:
No need for this column when we are on this page (the account page)

Reply:
Removed the account column in the user table, in the assets table I'm not sure. Should we maybe include the public assets in that view as well and keep the column? Let me know what the preferred change is.

Reply:
The solution Felix and I discussed is to put the public Assets in the account page as well. I've implemented this now. I think this is the best way to solve this in a general way for now.

Reply:
On "My Account"?

I don't understand what is wrong with my proposal.